qml/pages/CoverArtList.qml

```diff
@@ -66,7 +66,9 @@ BackgroundItem
     Connections {
         target: coversearch
         onImageChanged: {
-            if ((artist===dartist && album==dalbum) || (album===dartist && album==dalbum)) {
+            if ((artist===dartist && album==dalbum)
+                || (album===dartist && album==dalbum)
```

Comment:
**Not introduced by you**, you just enhanced the formatting:

Do you understand the logic of station `if (A or A)`, i.e. AFAICS the (now) line 70 is superfluous, correct?

Reply:
Well, there is a subtelty here, the (now) line 70 is slightly different than line 69. In line 69, it tries to match _both_ artist and album, while in line 70, it tries only tries to match the album with what has changed (assuming the artist being treated as the album). It's a bit strange, I'm sure @CepiPerez had a reason for this, but when filtering the model with artists only, `album` contains the artist and `artist` contains the number of albums of this artist. That's why I added the line 71, and maybe that's for such kind of reasons that line 70 ever exists.

Nota bene: without the addition of line 71, when you visit an artist page and search for a cover for this artist, when you come back to the artist page after having selected a cover, the artist cover is not updated.

Reply:
Oh, I read it three times, but did not see it the "subtelty" until now. :sad: Sometimes I am blind.